### PR TITLE
Revert stream button for moderators

### PR DESCRIPTION
**Pull request by DeeJayElly**
Revert stream button for moderators

### CODE REVIEW

File: react/features/toolbox/components/web/Toolbox.js [hunk 1]
```diff
@@ -989,6 +990,9 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
+            this._isModerator() && <LiveStreamButton
```
Comment (shapkarin) [inline react/features/toolbox/components/web/Toolbox.js]: Maybe better is merge develop and next use _isModerator prop form the my request that you looked at morning?

Comment (shapkarin) [reply react/features/toolbox/components/web/Toolbox.js]: In any case, please use `_isModerator` prop.

Comment (DeeJayElly) [reply react/features/toolbox/components/web/Toolbox.js]: Fixed

File: react/features/toolbox/components/web/Toolbox.js [hunk 2]
```diff
@@ -980,6 +981,9 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
+            this.props._isModerator && <LiveStreamButton
```
Comment (shapkarin) [inline react/features/toolbox/components/web/Toolbox.js]: it's already used as variable with destructuring assignment
https://github.com/aeternity/jitsi-meet/blob/7561cd7fd0ec6a74fba670cd8c40a7ac542bd384/react/features/toolbox/components/web/Toolbox.js#L957-L971

Comment (DeeJayElly) [reply react/features/toolbox/components/web/Toolbox.js]: Added

Comment (shapkarin) [reply react/features/toolbox/components/web/Toolbox.js]: what? it's already at `develop`

File: react/features/toolbox/components/web/Toolbox.js [hunk 3]
```diff
@@ -980,6 +981,9 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
+            this._isModerator && <LiveStreamButton
```
Comment (shapkarin) [inline react/features/toolbox/components/web/Toolbox.js]:
```suggestion
_isModerator && <LiveStreamButton
```

**Review by venimus — APPROVED** [review]

**Pull request merged**